src/test_workflow/benchmark_test/benchmark_test_suite.py

```diff
@@ -48,8 +48,10 @@ def __init__(
             user_tag = f"--user-tag=\"{args.user_tag}\""
             self.command += f" {user_tag}"
 
-        if args.capture_node_stat:
-            self.command += " --telemetry node-stats"
+        if args.telemetry:
+            self.command += " --telemetry "
+            for value in args.telemetry:
+                self.command += f"{value},"
```

Comment:
@tianleh This will append `,` at the end of the last telemetry option, have you verified that opensearch-benchmark wouldn't throw an error due to this?

Reply:
Yes, I verified that it works by running benchmark commands manually, please see the command in the description. 😄
Although it will have a tailing comma mark, the code can be concise.